### ISSUE TITLE
Uncaught Error: EPERM, operation not permitted '/Users/MKadlec/Library/Caches/TemporaryItems/Outlook Temp/#3681__Fixes_and_improvements.patch'

### ISSUE THREAD

**Issue by mkadlec** [atom/atom#5307]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.174.0
**System**: Mac OS X 10.10.1
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: EPERM, operation not permitted '/Users/MKadlec/Library/Caches/TemporaryItems/Outlook Temp/#3681__Fixes_and_improvements.patch'

```
At /Applications/Atom.app/Contents/Resources/app/src/workspace.js:504

Error: EPERM, operation not permitted '/Users/MKadlec/Library/Caches/TemporaryItems/Outlook Temp/#3681__Fixes_and_improvements.patch'
  at Error (native)
  at Object.fs.openSync (fs.js:455:18)
  at Object.module.(anonymous function) [as openSync] (/Applications/Atom.app/Contents/Resources/atom/common/lib/asar.js:422:20)
  at Project.module.exports.Project.open (/Applications/Atom.app/Contents/Resources/app/src/project.js:287:31)
  at Workspace.module.exports.Workspace.openURIInPane (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:492:31)
  at Workspace.module.exports.Workspace.open (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:418:19)
  at Ipc.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:45:32)
  at Ipc.emit (events.js:110:17)
  at process.<anonymous> (/Applications/Atom.app/Contents/Resources/atom/renderer/api/lib/ipc.js:22:29)
  at process.emit (events.js:118:17)

```

### Commands

```
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "grandson-of-obsidian"
    ]
  },
  "editor": {
    "fontSize": 14,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
color-picker, v1.2.5
git-log, v0.2.0
grandson-of-obsidian, v0.1.0
linter, v0.7.3
linter-js-yaml, v0.1.3
linter-ruby, v0.1.4
vim-mode, v0.17.0

# Dev
No dev packages
```

**Comment by mkadlec** [comment]
Crash. :(

**Comment by benogle** [comment]
This isnt a crash, right? It's just a red error, I assume.

**Comment by mkadlec** [comment]
It was a crash, I clicked a doc from outlook (on OSX Yosemite) and atom is my default editor.  Not super important, just wondered the cause.

**Comment by kevinsawicki** [comment]
I believe this was fixed by https://github.com/atom/atom/pull/5114 and released in Atom 0.175.0
